src/status_im/notifications/core.cljs

```diff
@@ -29,42 +29,86 @@
            (log/debug "notifications-denied")
            (re-frame/dispatch [:notifications.callback/request-notifications-permissions-denied {}]))))))
 
+(defn create-notification-payload
+  [{:keys [from to] :as payload}]
+  (if (and from to)
```

Comment:
we could introduce `valid-payload?` fn for this check `(and from to)` (though used only twice, not a big deal)

Reply:
done